docs/index.asciidoc

```diff
@@ -220,8 +220,9 @@ This will add a field named `kafka` to the logstash event containing the followi
   * Value type is <<string,string>>
   * Default value is `"true"`
 
-If true, periodically commit to Kafka the offsets of messages already returned by the consumer. 
-This committed offset will be used when the process fails as the position from
+If true, periodically commit to Kafka the offsets of messages already returned by the consumer.
+If value is `false` however, the offset is committed every time the consumer fetches the data from
+the topic. This committed offset will be used when the process fails as the position from
 which the consumption will begin.
```

Comment:
@amitavmohanty01 You did exactly what I suggested.  Unfortunately, I made a mistake with my suggestion. The description should go BEFORE the true/false information, like this:

> This committed offset will be used when the process fails as the position from which the consumption will begin.
> If true, periodically commit to Kafka the offsets of messages already returned by the consumer. If value is false however, the offset is committed every time the consumer fetches the data from the topic. 

Please accept my apologies for the misdirection.   If you would like to reorder the paragraph, I will approve it when you are done.  Or, I will reorder the sentences if you would prefer, since this is my mistake. Either way is fine.

This is a nice addition to the docs, and we appreciate your contribution. Thank you.

Reply:
Thanks for the feedback @karenzone I will modify and update.